src/PyodideWorker.js

```diff
@@ -98,6 +92,52 @@ const PyodideWorker = () => {
     await pyodide.loadPackagesFromImports(python);
 
     checkIfStopped();
+    await pyodide.runPythonAsync(
```

Comment:
I wonder if this might be the right time to start moving these into separate imports? (perhaps just focussing on this one for now)

Reply:
Yeah, possibly, I'd have to think more about this 🤔 Not sure if it'll be the same, but importing other stuff into the pyodide worker has not been particularly straightforward sadly